### DISCUSSION TITLE
Do we have to manually clean up docker running services?

### DISCUSSION THREAD

**Discussion by gmcamposano** [coollabsio/coolify#792] [Q&A] [answered]
Hello,
So I have been playing with coolify to understand how to use it and inspected the docker running processes. I found that the images of services and databases were still running in the background even though i have deleted them on the frontend ui. Do we have to manually delete the images that we are not using?

**Comment by peaklabs-dev** [comment, marked as answer]
In v4 there is automated cleanup
